pkg/auth/handler/webapp/verify_identity.go

```diff
@@ -78,10 +80,16 @@ func (h *VerifyIdentityHandler) GetData(r *http.Request, state *webapp.State, gr
 	}
 	var n VerifyIdentityNode
 	if graph.FindLastNode(&n) {
-		viewModel.IdentityDisplayID = n.GetVerificationIdentity().DisplayID()
+		rawIdentityDisplayID := n.GetVerificationIdentity().DisplayID()
 		viewModel.VerificationCodeSendCooldown = n.GetVerificationCodeSendCooldown()
 		viewModel.VerificationCodeLength = n.GetVerificationCodeLength()
 		viewModel.VerificationCodeChannel = n.GetVerificationCodeChannel()
+		switch authn.AuthenticatorOOBChannel(viewModel.VerificationCodeChannel) {
+		case authn.AuthenticatorOOBChannelSMS:
+			viewModel.IdentityDisplayID = phone.Mask(rawIdentityDisplayID)
+		default:
+			viewModel.IdentityDisplayID = rawIdentityDisplayID
```

Comment:
We also mask emails using `mail.MaskAddress`.

Reply:
OK, will add the corresponding mask as well